pkg/storage/unified/sql/db/dbimpl/db_engine.go

```diff
@@ -69,6 +67,47 @@ func getEngineMySQL(getter confGetter) (*xorm.Engine, error) {
 	return engine, nil
 }
 
+func configureTLS(getter confGetter, config *mysql.Config) error {
+	sslMode := getter.String("ssl_mode")
+
+	if sslMode == "skip-verify" || sslMode == "preferred" {
+		config.Params["tls"] = "preferred"
+	}
+
+	if sslMode == "true" {
```

Comment:
Hm, I don't know if you can simply use `true` here. IMO this is a super leaky configuration, but if you look at the [docs](https://github.com/search?q=repo%3Agrafana%2Fdeployment_tools+%22Critical+Webhook%22&type=code) `ssl_mode` wants the exact same parameters that you then pass to the driver. This means that depending on the driver, the parameter values change! `true` works for MySQL, but Postgres can take any of the [`libpq` arguments](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS).

Reply:
Have you seen that? https://github.com/grafana/grafana/blob/0854bd801b6cb35645dfbfe044bb031436a176a5/pkg/storage/unified/sql/db/dbimpl/db_engine.go#L113-L122

Reply:
I didn't notice this block was inside `getEngineMySQL`. My bad!